### PR TITLE
Admin pages

### PR DESCRIPTION
**Pull request by patjanawat**
- Create an admin page (**Closes**)
- Create on allowance component  (**Closes**)
- Create on distribution config component  (**Closes**)

Related to https://github.com/mesg-foundation/atd/issues/48

### CODE REVIEW

File: frontend/pages/admin/index.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          <DistributionConfigure />
+        </v-card-text>
+      </v-card>
+    </v-tab-item>
+  </v-tabs>
+</template>
+
+<script>
+import Allowance from "~/components/admin/allowance.vue";
+import DistributionConfigure from "~/components/admin/distributionConfig.vue";
```
Comment (NicolasMahe) [inline frontend/pages/admin/index.vue]: Be careful with the name!
```suggestion
import DistributionConfig from "~/components/admin/distributionConfig.vue";
```

File: frontend/pages/admin/index.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-tabs left color="cyan" dark icons-and-text>
+    <v-tabs-slider color="yellow"></v-tabs-slider>
```
Comment (NicolasMahe) [inline frontend/pages/admin/index.vue]: You don't need to create a variable `tabs` that contains the data in order to use `v-tab`.
You can simply put the data directly in the html, the data are hardcoded anyway and there is only 2 elements.
The simpler is the better even if there is a little bit of duplication.
Example:
```suggestion
    <v-tabs-slider color="yellow"></v-tabs-slider>
      <v-tab href="#allowances">
        Allowances <v-icon>attach_money</v-icon>
      </v-tab>
      <v-tab href="#distributionConfigs">
        Distribution Configs <v-icon> blur_on </v-icon>
      </v-tab>

     <v-tab-item value="allowances">
      <v-card flat>
        <v-card-text>
          <Allowance />
        </v-card-text>
      </v-card>
    </v-tab-item>

     <v-tab-item value="distributionConfigs">
      <v-card flat>
        <v-card-text>
          <DistributionConfigure />
        </v-card-text>
      </v-card>
    </v-tab-item>

  </v-tabs>
    
    
```

File: frontend/pages/admin/index.vue (hunk 1 above)
Comment (patjanawat) [reply frontend/pages/admin/index.vue]: Resolved. Thanks.

File: frontend/pages/admin/index.vue (hunk 2 above)
Comment (patjanawat) [reply frontend/pages/admin/index.vue]: Done, thanks.

**Pull request merged**